### ISSUE TITLE
Contact Page: Skip to Content is announced with NVDA but does not work

### ISSUE THREAD

**Issue by bnhowell** [projecthydra/sufia#3101]
### Descriptive summary
Related to Sufia 7.2

Contact Page: Skip to Content is announced with NVDA but does not work

### Expected behavior
Skip to content link is announced and actionable

### Actual behavior
Skip to content link is announced via NVDA but is not actionable

### Related work
#3100

**Comment by mjgiarlo** [comment]
@bnhowell move to hyrax?
